### PR TITLE
Remove jscs

### LINKED ISSUE #327: Remove now-deprecated JSCS
[Ref](https://medium.com/@markelog/jscs-end-of-the-line-bc9bf0b3fdb2#.8hl6oipg6)

### PR DESCRIPTION
**Pull request by TerrenceLJones**
This resolves #327.

### CODE REVIEW

**Comment by jmeas** [comment]
Nice! Thanks for the PR, @TerrenceLJones ! :v: I'll review this soon.

File: app/templates/.eslintrc [hunk 1]
```diff
@@ -1,8 +1,41 @@
...
+    "space-before-function-paren": [ 2, "never" ],
+    "space-infix-ops": 2,
+    "space-in-parens": [ 2, "never" ],
+    "space-unary-ops": [ 2, {
+        "nonwords": false,
+        "overrides": {}
+        }
+    ],
+   "strict": 0,
+    "valid-jsdoc": 2
```
Comment (jmeas) [inline app/templates/.eslintrc]: I'm a fan of many of these rules, but one thing I've been thinking is that I want the boilerplate to be more vanilla / customizable. Would you mind deleting errthang underneath `"rules"`?

**Comment by jmeas** [comment]
Looks great. I just had one comment. Make that change and we can get this meeeerged and released :v:

**Comment by TerrenceLJones** [comment]
@jmeas...question...these rules equate to what the jscs `preset:google` was. Would you rather replace these rules with the eslint Google [config](https://github.com/google/eslint-config-google) or still remove them all?

**Comment by jmeas** [comment]
Still removed, I think, so that folks can decide for themselves what rules they want to use.

**Comment by TerrenceLJones** [comment]
Sounds good, thanks!

**Comment by TerrenceLJones** [comment]
Alright @jmeas those rules are now removed. This PR should be ready to go! 🎱

**Comment by jmeas** [comment]
Nice! Thanks! I'll merge and release tonight.

**Comment by TerrenceLJones** [comment]
Awesome sounds good! 👍🏽

**Comment by jmeas** [comment]
Gonna fix the broken build over in #334. :v:

**Pull request closed without merging**